src/components/input/elastic-textarea.ts

```diff
@@ -12,6 +12,7 @@ export const createElasticTextArea = (textarea: HTMLTextAreaElement) => {
   const maxRows = 500; // Used to prevent infinite loop if textarea has fixed height
   const minRows = Number(textarea.getAttribute('rows')) || 2;
   let rows = Number(textarea.getAttribute('rows')) || minRows;
+  textarea.setAttribute('rows', String(rows));
```

Comment:
@spaceninja I wanted to flag this change, one of the tests fails without it (the test makes sure that when you initialize the JS, if there is no rows attribute, it sets it to 2). I'm fine with a different solution to that if you have one

Reply:
👍 I think that's fine — there's no difference for the user (because the update function runs a second later and will set the rows value), so this is just a sanity check for the tests.

Reply:
This does make a difference for the user. When the update function runs, in this case it does not set the rows value, if it did the test would pass without this addition.

`update` -> `isScrolling` is false -> `shrink()` -> `rows < minRows` so while loop never runs

Reply:
I see what you're saying that the initial update function doesn't add a rows value if one isn't there and the contents don't exceed the min value. But I don't understand how that affects the user. Will they see any changes in behavior as a result of the rows value being added during the create function?

Reply:
Yes. Since the default `minRows` value is `2`, with this addition, now when the JS is initialized it will update the textarea to 2 lines tall. Before this change, it would leave it at just 1 line

Reply:
I'm unable to reproduce that behavior, but what you're describing is fine — the textarea will be initialized with the `minRows` value if no `rows` attribute is provided.